.github/workflows/ci-release-gate.yml

```
@@ -38,180 +37,221 @@ jobs:
+  trivy-scan:
+    name: Trivy Scan (Digest Gate)
+    needs: [ build-push-frontend, build-push-backend, build-push-worker ]
+    if: false
+    permissions:
+      contents: read
+      packages: write # Used to push images to `ghcr.io` if used.
+      id-token: write # Needed to create an OIDC token for keyless signing
+      attestations: write # Needed to upload attestations to the GitHub registry      
+    uses: ./.github/workflows/trivy-release.yml
+    with:
+      image-digest-backend: ${{ inputs.build-push-backend.image-digest }}
```

actionlint (1.7.12)

[error] property "build-push-backend" is not defined in object type {}

(expression)